### ISSUE TITLE
Fix warning "Helm source <file>: after-init-hook Should be defined as a symbol

### ISSUE THREAD

**Issue by darim1** [emacs-helm/helm#2549]
### What happened?

I observe warnings, when trying to search my org files with helm-org-rifle:

```
Warning (emacs): Helm source ‘file1.org’: after-init-hook Should be defined as a symbol Disable showing Disable logging
Warning (emacs): Helm source ‘file2.org’: after-init-hook Should be defined as a symbol Disable showing Disable logging
...
```
The only after-init hook I employ is a key binding:

```
(add-hook 'helm-after-initialize-hook
          (lambda()
            (define-key helm-buffer-map (kbd "ESC") 'helm-keyboard-quit)
            (define-key helm-M-x-map (kbd "ESC") 'helm-keyboard-quit)
            (define-key helm-map (kbd "ESC") 'helm-keyboard-quit)))
```

I've already disabled it, with no luck.

### How to reproduce?

1. Install helm-org-rifle
2. define some agenda files
3. run helm-org-rifle
4. Observe the warnings, before the pattern can be entered

### Helm Version

Melpa or NonGnuElpa

### Emacs Version

Emacs-28/29

### OS

Windows

### Minimal configuration

- [X] I agree using a minimal configuration

**Comment by thierryvolpiatto** [comment]
darim1 ***@***.***> writes:

> What happened?
>
> I observe warnings, when trying to search my org files with helm-org-rifle:
>
> Warning (emacs): Helm source ‘file1.org’: after-init-hook Should be defined as a symbol Disable showing Disable logging
> Warning (emacs): Helm source ‘file2.org’: after-init-hook Should be defined as a symbol Disable showing Disable logging
> ...

The warning is explicit, please fix it in helm-org-rifle (or report it
to helm-org-rifle).

Thanks.


-- 
Thierry

**Comment by thierryvolpiatto** [comment]
Ping @alphapapa

**Comment by darim1** [comment]
Sorry and thanks!

**Comment by stefan2904** [comment]
For the record:

Here is the upstream report: alphapapa/org-rifle#78

And here the fix: alphapapa/org-rifle#77

As of 2022-10-17 the fix is not released, though. (Latest `helm-org-rifle` release is 20200512.1943 / 1.7.1)
